{"name": "test_add2[【类型】输入为空，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：和20.93开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660805207762, "stop": 1660805207763}], "attachments": [{"name": "log", "source": "38218e52-30a4-47c5-afc9-1a1a74811e6a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "9cace5d1-75b6-48ce-9f8d-1810d26e88db-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a3c66140-5786-4350-911e-97c76e7b13dc-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "''"}, {"name": "y", "value": "20.93"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660805207762, "stop": 1660805207763, "uuid": "e720c5a2-bba6-4dda-928c-1275a9bbc19a", "historyId": "bde00037f73fbb50b529755b33d90458", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "7552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}